File: help/pt_pt/exercicios_da_ficha_9/Ficha09_006/Ficha09_006.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Limite inferior : " );// print text
    inf = parseInt(prompt("Limite inferior : " ,"0") );
    
    document.write(inf + "\n");// print input 
    if(inf % 2 == 0 ){
        inf++;
    }
    document.write("Limite superior : " );// print text
    sup = parseInt(prompt("Limite superior : " ,"0") );
    
    document.write(sup + "\n");// print input 
    while(inf <= sup ){
        document.write(inf + " " );
        inf +=2;
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Thu, 2022 Sep 29 12:25

File: help/pt_pt/exercicios_da_ficha_9/Ficha09_006/Ficha09_006.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha09_006 {

    //Main Function
    static void Main() {
        Console.Write("Limite inferior : " );
        int inf = int.Parse(Console.ReadLine());
        if(inf % 2 == 0 ){
            inf++;
        }
        Console.Write("Limite superior : " );
        int sup = int.Parse(Console.ReadLine());
        while(inf <= sup ){
            Console.Write(inf + " " );
            inf +=2;
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Thu, 2022 Sep 29 12:25
